### ISSUE TITLE
Notebooks + doc require porting to Python 3

### ISSUE THREAD

**Issue by lkuchenb** [KohlbacherLab/epytope#12]
The notebooks are still Python 2 and won't run with the ported lib. The docs probably also have some Python 2 specific examples.

**Comment by jonasscheid** [comment]
I checked the issues Antonia addressed by comparing  the upcoming PR #42 against the legacy branch . I'll go over them step by step:
**1. CleavageAndTAPPrediction**

_Cell 4: The IDs and predicted of PCM differ from the ones in the legacy notebook. I find no IDs which match the one featured of the legacy notebook. I guess the names or data changed, but only by a tiny amount. (eg. -1.93 vs -1.77)_
Discrepancy probably happened after the [update of the proteins.fasta input file](https://github.com/KohlbacherLab/epytope/commit/30bf75b616f14706758220fc879416c1e651d80d). The current output makes more sense, because the right ID and Seqence is imported. 

_Cell 5: Again, marginally different values and ID compared to the legacy tutorial._
See comment above

_Cell 8: CleavageSitePredictorFactory using proteasmm_c returns marginally different values (eg. -1.93 vs -1.77)_
I don't observe that

_Cell 13: Legacy demonstrated the function filter_result with [("svmtap",ge, -30)]. I updated svmtap to smmtap and changed to filter greater 1 since the values a mostly in the range of -1 and 1, but maybe another value makes even more sense._
That is to be discussed. I arbitrarily inserted a filter number of 1 as Antonia suggested, but since it's a tutorial it is fine I guess.

_Cell 14: Legacy again used svmtap for the predictions here, changed it to smmtap. Choose a sensible threshold for smmtap so that the number of peptides after TAP transport changes. Preliminarily changed -30 to 1 but I am not sure if we want the highest scores for smmtap. Also, I don’t have UniTope installed, so this part is commented out and should be run again if UniTope wanted._
See comment above. UniTope is not supported anymore. Replaced by `smm`

_Cell 14:_ see comment above 

_Cell 15:_ Adjusted to new EpitopePredictionResult structure


**3. Epitope Prediction**
_Cell 4: read_fasta with type protein now returns the reference number contained in the file eg. “NP_852610.1” instead of as shown in legacy “Protein_0”; this is probably defined in Core/Protein_
Yes it is. Since the `proteins.fasta` file changed (as stated above) that changed as well.

_Cell 6: There are a lot more methods listed, this might depend on what methods I had installed, so it might make sense to rerun this with only the necessary methods installed. The following methods are missing compared to legacy: unitope 1.0 (not installed from me.) and svmhc (we removed this one.)_
New tools have been added eg netmhc family, therefore that is fine

_Cell 12: Returns NaNs for all methods (bimas, sim, syfpeithi) for me._
Not for me. The values are identical to the legacy branch

_Cell 13: Filter for a meaningful method and value. Updated filter to syfpeithi (was svmhc before, which was removed), but due to all values being NaN, returns empty table._
See comment above

_Cell 15: The value 1.0 is predicted for all while legacy returns the value 0 for (L, L, G, A, T, C, M, F, V) and (S, Y, F, P, E, I, T, H, I)_
It's random right


**5. HLA Typing**
Cannot execute this, because the pip way of installing `Optitype` is not working anymore (since pip doesn't support python2 anymore). Maybe I'm wrong here


**9. Vaccine Design**
_Cell 3: BIMAS is not working for me: “No predictions could be made with bimas for given input. Check your epitope length and HLA allele combination.”_
Worked for me


The tutorials have been updated accordingly in the PR #42
